File-Formats/DocIO/Working-with-Charts/Format-Legend.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Modify the appearance of legend | Syncfusion
+description: Learn how to modify the appearance of legend in a chart in a  Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Legend
+
+Legends are visual **pictorial** hints that provide a viewer information that helps them understand an chart. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library you can customize the **legend** in the chart.
```

Comment:
Legends are visual **pictorial** hints that provide a viewer information that helps them understand an chart. Using Syncfusion [.NET Core Word (DocIO)](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) library, you can customize the **legend** in the chart.